### PR TITLE
Shows

### PR DESCRIPTION
**Pull request by larsonkonr**

### CODE REVIEW

**Comment by mikedao** [comment]
Squash dem commits.

**Comment by larsonkonr** [comment]
NO!!!!!!

**Comment by mikedao** [comment]
Then it shouldn't be merged.

**Comment by mikedao** [comment]
Is this ready to be merged in? It's not labelled with anything.

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -7,5 +7,7 @@ def index
 
   def show
     @job = Job.find(params[:id])
+    @job_description =
```
Comment (bryceholcomb) [inline app/controllers/jobs_controller.rb:10]: What is going on here? Should we be formatting the information better before it hits the DB?

**Comment by larsonkonr** [comment]
I dont know how to use the labels

**Comment by mikedao** [comment]
@larsonkonr 
http://bit.ly/1OSlhCo

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (larsonkonr) [inline app/controllers/jobs_controller.rb:10]: When we get real data it will go away but thats the only way to format the seed data properly

**Pull request merged**